MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -45,7 +45,11 @@ Please see [Before Development](MODULE_EVALUATION_TEMPLATE#before-development) f
   * _This is not applicable to libraries_
 * Module descriptor MUST include interface requirements for all consumed APIs (3, 5)
   * _This is not applicable to libraries_
-* Third party dependencies use an Apache 2.0 compatible license (2)
+* Inclusion of third party dependencies complies with [ASF 3rd Party License Policy](https://apache.org/legal/resolved.html) (2)
+  * Uses README for [Category B Appropriately Labelled Condition](https://apache.org/legal/resolved.html#appropriately-labelled-condition)
```

Comment:
I am fine with using the README for the appropriately labeled conditions (since ASF uses the README location as the "example" location for that). I assume we can later add other libraries to this list as needed.